src/quacc/recipes/espresso/_base.py

```diff
@@ -75,7 +75,9 @@ def base_fn(
         **calc_flags,
     )
 
-    final_atoms = run_calc(atoms, copy_files=copy_files)
+    geom_file = template.outputname if template.binary == "pw" else None
+
+    final_atoms = run_calc(atoms, geom_file=geom_file, copy_files=copy_files)
```

Comment:
Had a look at run_calc(), maybe we even want to get all atoms using index = "::" so that the final result dict of the job contains all atoms object? This would be needed if the user run calculation = 'md' for example

Reply:
Good point. This hasn't come up before because in other recipes where I'm running internal relaxations (e.g. Vasp, ORCA, Gaussian, Q-Chem), there are custom output file parsers being called (e.g. pymatgen, cclib) that fetch that data automatically. But here, we are relying on data obtained solely from ASE, and so that would be worth adding. Opened an issue report in #1451.